core/src/entities/enrichments/vulnerabilities.rs

```diff
@@ -1,16 +1,24 @@
 use crate::entities::enrichments::cvss::Summary;
+use crate::entities::tasks::{Task, TaskRef};
+use crate::Error;
 use serde::{Deserialize, Serialize};
 use serde_with::skip_serializing_none;
 use std::fmt::{Display, Formatter};
 
-///
+/// Alias for the native CycloneDx Severity enum.
 pub type Severity = crate::entities::cyclonedx::Severity;
 
 /// Identified security issue for a [Package].
 #[derive(Clone, Debug, Deserialize, Serialize)]
 #[serde(rename_all = "camelCase")]
 #[skip_serializing_none]
 pub struct Vulnerability {
+    /// Unique identifier for instance.
+    pub id: String,
+
+    /// Package URL that the [Vulnerability] pertains to.
+    pub purl: String,
```

Comment:
It would be cool if this was one of these:

https://docs.rs/packageurl/latest/packageurl/

No worries if that isn't in the cards for this PR, but if you can make the change then that's great too.

Reply:
yeah. I think trying to address that now, will be a long delay and a rushed implementation.